### DISCUSSION TITLE
Custom Script - make the scheduled jobs editable

### DISCUSSION THREAD

**Discussion by jnc27** [netbox-community/netbox#16306] [Q&A]
I have a custom script, which I want to schedule multiple times. My problem is, that I cannot edit any scheduled jobs or see what has been selected in my formular. Is there any chance to make the scheduled jobs editable?

**Comment by llamafilm** [comment]
Related: https://github.com/netbox-community/netbox/discussions/14983, https://github.com/netbox-community/netbox/discussions/14989
